src/website/src/pages/ec/getting-started/index.md

```diff
@@ -62,39 +64,44 @@ New components are continuously being added to the library. The team is also con
 </html>
 ```
 
-First:
-
-<ul>
-  <li>
-    Replace <code>[ECL_VERSION]</code> by{' '}
-    <code>v{process.env.ECL_VERSION}</code> to use the latest version
-  </li>
-  <li>
-    For more security, use these SRI hashes:
-    <ul>
-      <li>
-        for ecl-ec-preset-website.css:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_CSS}</code>
-      </li>
-      <li>
-        for ecl-ec-preset-website-print.css:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_PRINT_CSS}</code>
-      </li>
-      <li>
-        for ecl-ec-preset-website.js:{' '}
-        <code>{process.env.ECL_EC_PRESET_WEBSITE_JS}</code>
-      </li>
-    </ul>
-  </li>
-</ul>
-
-Then:
-
-- copy the markup from the template you want and paste it in the `<body>`.
+### Update file path and name
+
+- If you wish to use ECL EU instead of ECL EC, you can find information here <Link to="/eu/getting-started/" label="Getting started EU" />
+- Replace <code>[ECL_VERSION]</code> by <code>v{process.env.ECL_VERSION}</code> to use the latest version
+- For more security, use the SRI hashes corresponding to the files. They all can be found in the [release page](https://github.com/ec-europa/europa-component-library/releases), in a json file, but here are the most common ones:
+
+<details>
+  <summary>SRI hashes (click to expand)</summary>
+  
+  <ul>
+    <li>for `ecl-ec.css`: <code>{process.env.ECL_EC_CSS}</code></li>
+    <li>for `ecl-ec-print.css`: <code>{process.env.ECL_EC_PRINT_CSS}</code></li>
+    <li>for `ecl-ec.js`: <code>{process.env.ECL_EC_JS}</code></li>
+  </ul>
+</details>
+
+### Optional files
+
+There are several optional files, which can be included in your site on not, depending on the needs: 
+
+- `ecl-reset.css`: contains some css reset rules, mostly based on normalize.css, with a few custom additions
+- `ecl-ec-default.css`: apply ECL styling to some default HTML tags (links, buttons, table, ...). Note that this file is meant to be used in specific cases, the recommanded way to use ECL is still to add corresponding classes to HTML elements
+
+<details>
+  <summary>SRI hashes for optional files (click to expand)</summary>
+  
+  <ul>
+    <li>for `ecl-reset.css`: <code>{process.env.ECL_RESET_CSS}</code></li>
+    <li>for `ecl-ec-default.css`: <code>{process.env.ECL_EC_DEFAULT_CSS}</code></li>
+  </ul>
+</details>
+
+### Additional information
+
+- to integrate ECL components, copy the markup from the template you want and paste it in the `<body>`.
```

Comment:
We had this also in v2, not really clear what is this template to copy the markup from, maybe we could be a bit more specific.
And since we release the twig templates using the `template` word might be confusing, at the same time we could mention the availability of the twig templates on npm as an existing implementation of the ECL components..